core-bundle/src/Resources/contao/library/Contao/Widget.php

```diff
@@ -37,6 +37,7 @@
  *     }
  *
  * @property string        $id                 The field ID
+ * @property string        $type               the field type
```

Comment:
If you add this here, you have to remove it in the `FormText` and `FormRange` classes:

https://github.com/contao/contao/blob/74c56f07e14a1ce2e0fda81c50abb066c6b4a7fa/core-bundle/contao/forms/FormText.php#L19

https://github.com/contao/contao/blob/74c56f07e14a1ce2e0fda81c50abb066c6b4a7fa/core-bundle/contao/forms/FormRange.php#L17

Reply:
Fixed in 7b431a6180f791b9c9bb3c48fec6a96096b6cb33